js/src/views/Dapps/dapps.css

```diff
@@ -25,6 +25,6 @@
 
 .item {
   display: inline-block;
-  margin: 0 0 0.25em;
-  width: 100%;
+  margin: 0.25%;
```

Comment:
Okay, fixed (nb: `0.125em` but for each which makes `0.25em` for a two column layout)